test/DurableTaskEndToEndTests.cs

```diff
@@ -416,7 +416,7 @@ public DurableTaskEndToEndTests(ITestOutputHelper output)
         {
             string[] orchestratorFunctionNames =
             {
-                nameof(TestOrchestrations.OrchestratorThrow)
+                nameof(TestOrchestrations.Throw)
```

Comment:
>Throw [](start = 42, length = 5)

Reverting the orchestrator name back to "Throw" to verify that activity and orchestrators with the same name can be resolved correctly.